run.go

```diff
@@ -71,11 +71,12 @@ const (
 )
 
 type RunRequest struct {
-	AssistantID  string         `json:"assistant_id"`
-	Model        *string        `json:"model,omitempty"`
-	Instructions *string        `json:"instructions,omitempty"`
-	Tools        []Tool         `json:"tools,omitempty"`
-	Metadata     map[string]any `json:"metadata,omitempty"`
+	AssistantID            string         `json:"assistant_id"`
+	Model                  *string        `json:"model,omitempty"`
+	Instructions           *string        `json:"instructions,omitempty"`
+	AdditionalInstructions *string        `json:"additional_instructions,omitempty"`
```

Comment:
Hey, thank you for the PR! Could we please use normal `string` instead of `*string` in all three cases here?

Reply:
Done.

Reply:
Note there are still other uses of `*string` JSON fields:

```
assistant.go:	Name         *string         `json:"name,omitempty"`
assistant.go:	Description  *string         `json:"description,omitempty"`
assistant.go:	Instructions *string         `json:"instructions,omitempty"`
assistant.go:	Name         *string         `json:"name,omitempty"`
assistant.go:	Description  *string         `json:"description,omitempty"`
assistant.go:	Instructions *string         `json:"instructions,omitempty"`
assistant.go:	LastID     *string     `json:"last_id"`
assistant.go:	FirstID    *string     `json:"first_id"`
edits.go:	Model       *string `json:"model,omitempty"`
error.go:	Param          *string     `json:"param,omitempty"`
messages.go:	AssistantID *string          `json:"assistant_id,omitempty"`
messages.go:	RunID       *string          `json:"run_id,omitempty"`
messages.go:	FirstID *string `json:"first_id"`
messages.go:	LastID  *string `json:"last_id"`
```